Cronos/cronos-evm-starter-via-rpc/src/mappings/mappingHandlers.ts

```diff
@@ -20,12 +20,11 @@ type ApproveCallArgs = [string, BigNumber] & {
 export async function handleEthermintEvmEvent(
   event: EthermintEvmEvent<TransferEventArgs>
 ): Promise<void> {
-  logger.info(JSON.stringify(event.args.value));
   const transaction = Transaction.create({
     id: event.transactionHash,
-    value: event.args.value.toBigInt(),
```

Comment:
According to the abi these should be `wad` `src` dist`. The type at the top is wrong 

https://github.com/subquery/cosmos-subql-starter/blob/main/Cronos/cronos-evm-starter-via-rpc/erc20.abi.json#L50